scripts/patchFileDeps.js

```diff
@@ -0,0 +1,30 @@
+// pnpm treats file: protocols as link:, so it doesn't copy the actual files
+// into node_modules, causing Vite to still consider those deps linked.
+// This script is called from postinstall hooks in playground packages that
+// uses the file: protocol, and copies the file: deps into node_modules.
```

Comment:
I think this issue is about a similar use case: https://github.com/pnpm/pnpm/issues/3510

Reply:
It does sound similar. I also tried using `.npmrc` inside a sub-package with [package-import-method](https://pnpm.io/npmrc#package-import-method) but it didn't seem to work.